data/sql/updates/pending_db_world/rev_1616252675947625100.sql

```diff
@@ -0,0 +1,521 @@
+(20753, 0, 0, 0, 4, 0, 100, 1, 0, 0, 0, 0, 11, 12493, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Dorgok - On Aggro - Cast \'12493\' (No Repeat)'),
+(20753, 0, 1, 0, 2, 0, 100, 1, 10, 30, 0, 0, 11, 39119, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Dorgok - Between 10-30% Health - Cast \'39119\' (No Repeat)'),
+(20753, 0, 2, 0, 0, 0, 100, 0, 3000, 5000, 18000, 19000, 11, 11962, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Dorgok - In Combat - Cast \'11962\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 19956;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 19956);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(19956, 0, 0, 0, 11, 0, 100, 1, 0, 0, 0, 0, 11, 35777, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Lookout - On Respawn - Cast \'35777\' (No Repeat)'),
+(19956, 0, 1, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 35240, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Lookout - Between 20-80% Health - Cast \'35240\' (No Repeat)'),
+(19956, 0, 2, 0, 2, 0, 100, 1, 5, 30, 0, 0, 11, 37786, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Lookout - Between 5-30% Health - Cast \'37786\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 19948;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 19948);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(19948, 0, 0, 0, 6, 0, 100, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Just Died - Say Line 1'),
+(19948, 0, 1, 2, 4, 0, 35, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Aggro - Say Line 0'),
+(19948, 0, 2, 0, 61, 0, 100, 1, 0, 0, 0, 0, 11, 34932, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Aggro - Cast \'34932\''),
+(19948, 0, 3, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 34802, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - Between 20-80% Health - Cast \'34802\' (No Repeat)'),
+(19948, 0, 4, 0, 2, 0, 100, 1, 5, 30, 0, 0, 11, 37786, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - Between 5-30% Health - Cast \'37786\' (No Repeat)');
```

Comment:
It already has AI

Reply:
I don't know if it's better than it is now

Reply:
Yeah, those i updated had kinda bad AI.
Overlaping with spells mostly.